pyproject.toml

```diff
@@ -102,6 +102,7 @@ export = [
     "openvino>=2024.0.0", # OpenVINO export
     "tensorflow>=2.0.0", # TF bug https://github.com/ultralytics/ultralytics/issues/5161
     "tensorflowjs>=3.9.0", # TF.js export, automatically installs tensorflow
+    "tensorstore==0.1.59; platform_machine == 'aarch64'", # stop falling back to check for older versions
```

Comment:
This might be brittle to future package combinations so we want to avoid this, but do you know why this is happening?

Reply:
Here is an example of the behavior affected by the CI also:
![image](https://github.com/ultralytics/ultralytics/assets/20147381/3da8420f-a34d-4326-8b75-4ed9a406faa0)

It is just about the time which is affecting this. It is not throwing any errros. Without fixing version to "0.1.59" Install requirements will take about 10min and after fixing version to "0.1.59", it only takes 3min.

Reply:
@glenn-jocher On Docker the diference is 12min vs 40min:
https://github.com/ultralytics/ultralytics/actions/runs/9291148517/job/25569028181
https://github.com/ultralytics/ultralytics/actions/runs/9290975309/job/25568471234

Reply:
@lakshanthad agreed, let's leave as is! I checked the other runners on main branch CI and they are all also using 0.1.59 (but they don't search for it, they somehow know to install it the first time), so this pinned version seems suitable.